src/WriterImpl.h

```diff
@@ -36,7 +36,7 @@ namespace e57
    class WriterImpl
    {
    public:
-      WriterImpl( const ustring &filePath, const ustring &coordinateMetaData );
+      WriterImpl( const ustring &filePath, const ustring &coordinateMetaData, const ustring &guid );
```

Comment:
This should take `WriterOptions` instead of `coordinateMetaData` now, shouldn't it?

Reply:
Done.